Firebase/InstanceID/FIRInstanceID.m

```diff
@@ -122,6 +122,9 @@ @interface FIRInstanceID ()
 @property(atomic, strong, nullable)
     FIRInstanceIDCombinedHandler<NSString *> *defaultTokenFetchHandler;
 
+/// A cached value of FID. Should be used only for `-[FIRInstanceID appInstanceID:]`.
```

Comment:
btw I don't see this firebaseInstallationsID is read anywhere, so what's the point of caching here?

Reply:
It's read in another file [here](https://github.com/firebase/firebase-ios-sdk/blob/0831e2d677be376495fe1fcb8fbb0e282afc505b/Firebase/InstanceID/FIRInstanceID%2BPrivate.m#L44)